frontend/src/static/js/v_zoom.js

```diff
@@ -8,7 +8,7 @@ window.vZoom = {
   template: window.$('v_zoom').innerHTML,
   data() {
     return {
-      filterTimeFrame: window.hashParams.timeframe,
+      filterTimeFrame: this.info.timeFrame ? this.info.timeFrame : window.hashParams.timeframe,
```

Comment:
can just pass the `timeFrame` at `openTabZoom` in `v_summary`?

Reply:
Though zoomfromramp is removed, I still need a mean to distinguish the two for the `getSliceColor` function. As the color is calculated using date and we want same color for all commit ramps to look like the same week/day ramp, when the zoom is opened by clicking ramp with week granularity, we need to use sdate instead of date in the  `getSliceColor`.

Reply:
ya, sorry under wrong section

Reply:
I'm sorry I don't quite get your point.

Why is zoom panel reactive to the change in granularity in summary? I thought they are independent? i.e. after opening zoom tab, changing the granularity in summary won't affect the zoom tab whatsoever?

Reply:
They are not independent. Change granularity in summary will change display in zoom. You can try preview of other PR.

Reply:
Ahhh I see. Ok noted.

Reply:
@fzdy1914 PR #830 is currently allowing zoom tab to be restorable after reloading, which requires the zoom tab and summary to be completely independent. So should we completely make them independent here? i.e. changing the filterTimeFrame in summary wont affect the zoom tab.

Reply:
let's put this on hold until #967 is merged, as it affects the implementation of this PR